### PR TITLE
New package: w3m

### PR DESCRIPTION
**Pull request by ronin-gw**

### CODE REVIEW

**Review by tldahlgren — APPROVED** [review]
LGTM *except* the validation issue.

File: var/spack/repos/builtin/packages/w3m/package.py [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+    depends_on('imlib2@1.0.5:', when='imagelib=imlib2 +image')
+
+    # fix for modern libraries
+    patch('fix_redef.patch')
+    patch('fix_gc.patch')
+
+    def _add_arg_for_variant(self, args, variant, choices):
+        for avail_lib in choices:
+            if self.spec.variants[variant].value == avail_lib:
+                args.append('--with-{}={}'.format(variant, avail_lib))
```
Comment (tldahlgren) [inline var/spack/repos/builtin/packages/w3m/package.py]:
```suggestion
                args.append('--with-{0}={1}'.format(variant, avail_lib))
```

Comment (tldahlgren) [reply var/spack/repos/builtin/packages/w3m/package.py]: This change will address at least part of the validation error.

**Review by tldahlgren — DISMISSED** [review]
Please resolve the validation check issue.

**Comment by ronin-gw** [comment]
@tldahlgren Thank you for your suggestion. I confirmed that the latest commit passed the `vermin` validation.

**Review by alalazo — APPROVED** [review]

**Pull request merged**